src/docfx/cli/Program.cs

```diff
@@ -44,15 +44,18 @@ internal static class Program
             {
                 try
                 {
+                    var startTime = DateTime.UtcNow;
```

Comment:
why not use Log?

Reply:
Do you mean `Log.Meaure?`

The `Done` message could take an additiona summary like # of errors and warnings, the output is different from measure I guess.

---
In reply to: [198065038](https://github.com/dotnet/docfx/pull/2942#discussion_r198065038) [](ancestors = 198065038)